src/SiteEnrollment/GrowerInformation.js

```diff
@@ -544,9 +550,61 @@ const saveNewGrowerAndFetchProducerId = async (enrollmentData = {}) => {
   });
 };
 
+const updateGrowerInfo = async (enrollmentData = {}) => {
+  let dataObject = {
+    firstName: enrollmentData.growerInfo.firstName,
+    lastName: enrollmentData.growerInfo.lastName,
+    email: enrollmentData.growerInfo.email,
+    phone: enrollmentData.growerInfo.phone
+      .split("(")
+      .join("")
+      .split(")")
+      .join("")
+      .split(" ")
+      .join("")
+      .split("-")
+      .join(""),
+    year: enrollmentData.year,
+    affiliation: enrollmentData.affiliation,
+    collaborationStatus: enrollmentData.growerInfo.collaborationStatus,
+  };
+  let dataString = qs.stringify(dataObject);
+  return await Axios.post(`${apiURL}/api/growers/add`, dataString, {
+    auth: {
+      username: apiUsername,
+      password: apiPassword,
+    },
+    headers: {
+      "content-type": "application/x-www-form-urlencoded;charset=utf-8",
+    },
+  });
+};
+
+  const updateSite = (enrollmentData = {}, getTokenSilently, code, producerId, year) => {
+          let newGrowerPromise = updateGrowerInfo({...enrollmentData, year: year, affiliation: code});
```

Comment:
it seems like this should only be called if you are adding a new producer, but this is getting called even if you're using an existing producer correct?